js/src/widgets/imageView.js

```diff
@@ -528,117 +634,104 @@
       }
     },
 
-    createOpenSeadragonInstance: function(imageUrl) {
-      var infoJsonUrl = imageUrl + '/info.json',
-      uniqueID = $.genUUID(),
-      osdID = 'mirador-osd-' + uniqueID,
-      infoJson,
-      _this = this;
-
-      this.element.find('.' + this.osdCls).remove();
+    initialiseImageCanvas: function() {
+      var _this = this,
+          osdID = 'mirador-osd-' + $.genUUID(),
+          canvasModel = _this.canvases[_this.canvasID];
 
-      jQuery.getJSON(infoJsonUrl).done(function (infoJson, status, jqXHR) {
-        _this.elemOsd =
-          jQuery('<div/>')
+      _this.elemOsd =
+        jQuery('<div/>')
         .addClass(_this.osdCls)
         .attr('id', osdID)
         .appendTo(_this.element);
 
-        _this.osd = $.OpenSeadragon({
-          'id':           osdID,
-          'tileSources':  infoJson,
-          'uniqueID' : uniqueID
-        });
+      _this.osd = OpenSeadragon({
+        id: osdID,
+        uniqueID: osdID,
+        preserveViewport: true,
+        blendTime: 0.1,
+        alwaysBlend: false,
+        showNavigationControl: false
```

Comment:
This is using the OpenSeadragon object directly, since the way it is used differs somewhat from the base version. These can be synchronised and we bring the other views into the same logic.